src/renderer/renderer.ts

```
@@ -199,13 +190,8 @@ function setupEventListeners(): void {
   // Type filter buttons
   typeFilter.querySelectorAll('.type-toggle').forEach((btn) => {
     btn.addEventListener('click', () => {
-      selectedType = (btn as HTMLElement).dataset.type as
-        | 'All'
-        | 'cask'
-        | 'formula';
-      document
-        .querySelectorAll('.type-toggle')
-        .forEach((b) => b.classList.remove('active'));
+      selectedType = (btn as HTMLElement).dataset.type as 'All' | 'cask' | 'formula';
+      document.querySelectorAll('.type-toggle').forEach((b) => b.classList.remove('active'));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)